lib/version_utils.pm

```diff
@@ -289,5 +289,6 @@ sub is_system_upgrading {
 }
 
 sub is_pre_15 {
-    return (is_sle && !sle_version_at_least('15')) || (is_leap && !leap_version_at_least('15.0')) || !is_tumbleweed;
+    return 0 if is_tumbleweed;
+    return is_sle('<15') || is_leap('<15.0') ;
```

Comment:
is_leap doesn't support new functionality we have for is_sle. Still have to use leap_version_at_least here

Reply:
It was implemented last week, also I asked you in the ticket, but I have to re-run the verification to show you the behaviour